sample/kotlin/src/main/kotlin/com/datadog/android/sample/home/HomeFragment.kt

```diff
@@ -57,6 +57,7 @@ internal class HomeFragment :
             R.id.navigation_logs -> R.id.fragment_logs
             R.id.navigation_crash -> R.id.fragment_crash
             R.id.navigation_traces -> R.id.fragment_trace
+            R.id.navigation_otel_traces -> R.id.fragment_otel_traces
```

Comment:
I think having them in 2 different screens can be confusing. We need to have all traces on the same screen under 2 different groups (called OpenTracing and OTel), or rename simple "Traces" to "OpenTracing Traces", because it is not clear what is the difference between traces and Otel traces.

Reply:
it will look a bit messy and overloaded that screen don't you think ?

Reply:
Let's have 2 different screen then, but rename "Traces" to "OpenTracing Traces" or something like that to make it clear what is the difference between these 2 screens.